samples/UnitTesting/package.json

```diff
@@ -19,6 +20,7 @@
   },
   "devDependencies": {
```

Comment:
> I didn't want to create the test app from scratch (mostly because it seemed more complicated) and preferred to just copy from the samples.

My preference is to have a separate test app in the `test` folder. I think a sample app will always have a different purpose than a test app and so we will run into conflicting goals pretty quickly. For example, for the purposes of https://github.com/Azure/azure-functions-durable-js/issues/353, we don't want _any_ `@types/` packages listed in the package.json otherwise it could hide a missing dependency error in `durable-functions`. But for a sample app you _would_ want to have `@types/` packages because that's what all users want in real life

When I think of a "smoke test", I think of a simple hello world app. I would suggest you use VS Code or core tools to create a project and add all three of the durable function triggers to create the simplest "working" app possible. Here's a tutorial if you want to follow exactly what a user would likely do: https://docs.microsoft.com/azure/azure-functions/durable/quickstart-js-vscode (Just use TypeScript instead of JavaScript and you can skip everything after you create the projects/functions)

Reply:
Created a new test app in the test folder and using that for the smoke test now